### ISSUE TITLE
Pipe history / automatic backups

### ISSUE THREAD

**Issue by anewuser** [pipes-digital/pipes#117]
Another feature idea: save the last few edits that a user has done to a pipe. In case he/she needs to revert one of the changes, it'll be possible to pick it from the history.

This can be another premium pipes.digital feature to motivate users to upgrade their accounts.

**Comment by onli** [comment]
Good idea and quite doable. Could have some slots in the DB per pipe for old versions.
